apps/admin-client/src/styles.css

```
@@ -0,0 +1,370 @@
+  --color-sidebar-accent-foreground: var(--sidebar-accent-foreground);
+  --color-sidebar-border: var(--sidebar-border);
+  --color-sidebar-ring: var(--sidebar-ring);
+}
+
+html,
+body,
+#app {
+  min-height: 100%;
+}
+
+/*body {*/
```

Stylelint (17.6.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)